src/display/canvas.js

```diff
@@ -1095,8 +1103,13 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
             if (height === 0) {
               height = this.getSinglePixelWidth();
```

Comment:
Nit: It might perhaps be a tiny bit more efficient to avoid extra function calls here, and in the `width === 0` case just above, by utilizing the now already available `pixelSize` value instead.